src/SPARQLStore/Connector/FusekiHttpRepositoryConnector.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace SMW\SPARQLStore\Connector;
+
+use SMW\SPARQLStore\Exception\BadHttpDatabaseResponseException;
+use SMW\SPARQLStore\QueryEngine\RepositoryResult;
+use SMW\SPARQLStore\QueryEngine\XmlResponseParser;
+
+/**
+ * @see https://jena.apache.org/documentation/serving_data/index.html
+ *
+ * @license GNU GPL v2+
+ * @since 2.0
```

Comment:
Well, only the name changed (git thought it is new instead of moved, go figure why!)